components/gfx/display_list/mod.rs

```diff
@@ -671,14 +663,11 @@ pub struct TextDisplayItem {
     /// The range of text within the text run.
     pub range: Range<ByteIndex>,
 
+    pub baseline_origin: LayoutPoint,
+    pub glyphs: Vec<GlyphInstance>,
+    pub font_key: FontInstanceKey,
     /// The color of the text.
     pub text_color: ColorF,
-
-    /// The position of the start of the baseline of this text.
```

Comment:
Why eliminate this documentation? I think it makes sense to preserve this as well as adding documentation for the new struct members.